tools/python/influx-migration/README.md

```diff
@@ -0,0 +1,642 @@
+
+```shell
+python3 influx_migration.py --full --src-host http://localhost:8086 --dest-host http://dest-server-address:8086 --csv --dest-org MyOrg
+
+```
+
+Output
+
+```
+INFO: influx_migration.py: Backing up bucket data and metadata using the InfluxDB v2 API
+INFO: influx_migration.py: Restoring bucket data and metadata from csv
+INFO: influx_migration.py: Restoring bucket some-bucket
+INFO: influx_migration.py: Restoring bucket another-bucket
+INFO: influx_migration.py: Restoring bucket primary-bucket
+INFO: influx_migration.py: Migration complete
+```
+
+### Full Migration Using CSV, Destination Organization, and S3 Bucket
+
+**Scenario**: Same as the previous example but using Linux or Mac and storing the files in the S3 bucket, my-s3-bucket. This avoids backup files overloading the local storage capacity.
+
+```shell
+python3 influx_migration.py --full --src-host http://localhost:8086 --dest-host http://dest-server-address:8086 --csv --dest-org MyOrg --s3-bucket my-s3-bucket
+```
+
+Output
+
+```
+INFO: influx_migration.py: Creating directory influxdb-backups
+INFO: influx_migration.py: Mounting influxdb-migration-bucket
+INFO: influx_migration.py: Creating directory influxdb-backups/my-s3-bucket/influxdb-backup-1698352128323
+INFO: influx_migration.py: Backing up bucket data and metadata using the InfluxDB v2 API
+INFO: influx_migration.py: Restoring bucket data and metadata from csv
+INFO: influx_migration.py: Restoring bucket some-bucket
+INFO: influx_migration.py: Restoring bucket another-bucket
+INFO: influx_migration.py: Restoring bucket primary-bucket
+INFO: influx_migration.py: Migration complete
+INFO: influx_migration.py: Unmounting influxdb-backups
+INFO: influx_migration.py: Removing temporary mount directory
+```
+
+## Security
+
+### Sensitive Data
+
+All data migrated by the Influx migration script, including sensitive data, is the responsibility of the user. Users are encouraged to be mindful about what data they choose to migrate and where that data will be migrated.
+
+### Permissions
+
+InfluxDB users should be granted least-privilege permissions and tokens granted to specific users, instead of operator tokens, should be used during migration.
+
+AWS-managed InfluxDB uses IAM permissions to control user permissions, least-privilege permissions should be granted to users.
+
+[//]: # "TODO: Once documentation or access to an instance is available list IAM permissions"
```

Comment:
We need to test and update this!

Reply:
This section now has all required permissions for the InfluxDB tokens. These permissions were already listed in the "Tokens" section, this section now links to the "Permissions" section instead of listing the permissions.